bin/publish

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Add the NPM token to the .npmrc file
+echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> "$HOME/.npmrc"
+
+# Publish the package, but only if the current published version is behind the latest version.
+PUBLISHED_VERSION=$(yarn info --json | jq -r '.data."dist-tags".latest')
+PACKAGE_VERSION=$(jq -r '.version' package.json)
+
+if [ "$PACKAGE_VERSION" != "$PUBLISHED_VERSION" ]; then
+  yarn publish || 0
```

Comment:
Do we want to have a `tag` or explicitly set this repo as `private`? Looking through the documentation for `yarn publish` I am not certain if we need anything else. https://yarnpkg.com/lang/en/docs/cli/publish/

Reply:
Good question. When you run `yarn version`, it automatically generates a tag for you with the version number. The readme instructions show how to push tags as well.